src/common/areaCard/selectedResource/useTransitionLogic.ts

```diff
@@ -24,13 +24,17 @@ const useTransitionLogic = ({ id, handleRemove, moveDown, moveUp }: UseTransitio
     setChanged('nothing');
   };
 
-  const doMoveUp = () => {
-    setChanged('up');
-  };
-
-  const doMoveDown = () => {
-    setChanged('down');
-  };
+  const doMoveUp = moveUp
+    ? () => {
+        setChanged('up');
+      }
+    : undefined;
```

Comment:
This seems a little bit unexpected to me. Couldn't this cause a need for the users of this hook to make checks before calling this function? Does this variable do two things: it's a flag ("moving up is possible") and a handler at the same time?

I don't think that it needs to be changed.